pkg/vere/vere.h

```diff
@@ -321,6 +330,7 @@
         c3_o       pep_o;                   //  prep for upgrade
         c3_o       play_o;                  //  replay event log and exit
         c3_i       xit_i;                   //  exit code for shutdown
+        u3_trac    tra_u;                   //  tracing information
```

Comment:
This struct member and the declaration can be (re)removed, as the tracing module has been refactored.